public/app/features/dashboard/state/PanelModel.ts

```diff
@@ -334,7 +334,7 @@ export class PanelModel implements DataConfigSource {
   pluginLoaded(plugin: PanelPlugin) {
     this.plugin = plugin;
 
-    if (plugin.panel && plugin.onPanelMigration) {
+    if (plugin.onPanelMigration) {
```

Comment:
The real change here is to allow migrations on angular panels *and* react panels